queuebot/cogs/queue/__init__.py

```diff
@@ -284,7 +291,12 @@ def has_emoji_slots(self, guild: discord.Guild) -> bool:
                             name=emoji_name, image=await resp.read(), reason='temp blob for vs'
                         ))
 
-            emote_sequence = " \N{SQUARED VS} ".join(map(str, temp_emotes))
+            if self.config.verbose_vs:
+                mapped_lines = [NUMERIC_EMOTES[temp_emotes.index(e)] + str(e) for e in temp_emotes]
```

Comment:
`enumerate` could be used here instead of indexing. By also using `zip`, `map`, and `str.join`, you could probably avoid using listcomps or lambdas entirely.